src/tooltip/src/Tooltip.js

```diff
@@ -118,12 +117,9 @@ const Tooltip = memo(props => {
     setIsShownByTarget(true)
   }
 
-  let shown = (propIsShown || isShown || isShownByTarget) && !isPopoverShown()
-
-  // Tooltip was explicitly set to not be shown
-  if (propIsShown === false) {
-    shown = false
-  }
```

Comment:
Actually now that I'm looking at this again I think this is meant to only capture when `isShown` is explicitly false. When it's set to `true` we do indeed need `&& !isPopoverShown()` as well.

Maybe revert my suggestion 😬

Reply:
lol okay